guardian/pinboard
# Update GraphQL dependencies to latest

## What does this change?
This PR updates the GraphQL libraries to the latest versions.

The primary aim of this update is to get rid of some Dependabot security alerts, and to lower technical debt going forward.

## How to test
+ Run `yarn graphql-refresh` locally - no errors should appear in the console.
+ Run the PR in the CODE environment to see if the new system brakes any CI steps.
+ Test in CODE - everything should operate as before.
+ (Are there any other tests we can perform?)
